protected/humhub/widgets/bootstrap/Button.php

```diff
@@ -223,6 +223,28 @@ public function confirm(
         return $this;
     }
 
+    /**
+     * Sets the button as disabled
+     */
+    public function disabled(bool $disabled = true): static
+    {
+        if ($disabled) {
+            $this->options['disabled'] = true;
```

Comment:
Should only be added if a button.
See https://getbootstrap.com/docs/5.3/components/buttons/#disabled-state -> "<a>s don’t support the disabled attribute"